### PR TITLE
i18n(fr): add `css-and-tailwind.mdx`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR adds `showcase.mdx` in french.

### CODE REVIEW

File: docs/src/content/docs/fr/guides/css-and-tailwind.mdx
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/css-and-tailwind.mdx:269]: An official reference for keeping "Chroma" https://fr.wikipedia.org/wiki/Chrominance

File: docs/src/content/docs/fr/guides/css-and-tailwind.mdx
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/css-and-tailwind.mdx:219]: Some references for the translation of "accent"

- https://developer.mozilla.org/fr/docs/Glossary/Accent
- https://learn.microsoft.com/fr-fr/windows/apps/design/signature-experiences/color#accent-color

**Review by delucis — APPROVED** [review]
LGTM! I’ll let you decide if you’d like to wait for a French language review or are happy with just my eyes checking this.

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Comment by HiDeoo** [comment]
> I’ll let you decide if you’d like to wait for a French language review or are happy with just my eyes checking this.

Well, I guess this did not take long ^^

Thanks you both for the reviews :raised_hands:

**Pull request merged**
